core/src/miner/mem_pool.rs

```diff
@@ -841,7 +841,6 @@ impl MemPool {
                 current_size < size_limit
             })
             .collect();
-
```

Comment:
Please do not add unrelated code change in a commit.

Reply:
Please review your commits in a PR line by line not to include unrelated changes.
You must fix these issues before request a review.

If removing these changes in your commit is difficult, please request help to me before request a review.